samples/cpp/benchmark_app/infer_request_wrap.hpp

```diff
@@ -33,44 +33,50 @@ class InferReqWrap final {
 
     ~InferReqWrap() = default;
 
-    explicit InferReqWrap(InferenceEngine::ExecutableNetwork& net, size_t id, QueueCallbackFunction callbackQueue)
-        : _request(net.CreateInferRequest()),
+    explicit InferReqWrap(ov::runtime::CompiledModel& net, size_t id, QueueCallbackFunction callbackQueue)
+        : _request(net.create_infer_request()),
           _id(id),
           _lat_group_id(0),
           _callbackQueue(callbackQueue),
           outputClBuffer() {
-        _request.SetCompletionCallback([&]() {
+        _request.set_callback([&](std::exception_ptr ptr) {
+            // TODO: Add exception ptr rethrow in proper thread
             _endTime = Time::now();
             _callbackQueue(_id, _lat_group_id, getExecutionTimeInMilliseconds());
         });
     }
 
     void startAsync() {
         _startTime = Time::now();
-        _request.StartAsync();
+        _request.start_async();
     }
 
     void wait() {
-        _request.Wait(InferenceEngine::InferRequest::RESULT_READY);
+        _request.wait();
     }
 
     void infer() {
         _startTime = Time::now();
-        _request.Infer();
+        _request.infer();
         _endTime = Time::now();
         _callbackQueue(_id, _lat_group_id, getExecutionTimeInMilliseconds());
     }
 
-    std::map<std::string, InferenceEngine::InferenceEngineProfileInfo> getPerformanceCounts() {
-        return _request.GetPerformanceCounts();
+    std::vector<ov::runtime::ProfilingInfo> getPerformanceCounts() {
+        return _request.get_profiling_info();
     }
 
-    InferenceEngine::Blob::Ptr getBlob(const std::string& name) {
-        return _request.GetBlob(name);
+    void setShape(const std::string& name, const ov::Shape& dims) {
+        // TODO check return status
+        _request.get_tensor(name).set_shape(dims);
     }
 
-    void setBlob(const std::string& name, const InferenceEngine::Blob::Ptr& data) {
-        _request.SetBlob(name, data);
+    ov::runtime::Tensor getBlob(const std::string& name) {
```

Comment:
```suggestion
    ov::runtime::Tensor get_tensor(const std::string& name) {
```

Reply:
done. for now it is kept camelCase for consistency with other functions. will be renamed later to _ case along with other functions if needed